independent-publisher-connectors/Line Message/apiDefinition.swagger.json

```diff
@@ -0,0 +1,159 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "Line Message",
+    "description": "The connector allows you to send messages to registered LINE bot and associated users.",
+    "contact": {
+      "name": "Felaray Ho",
+      "url": "https://github.com/felaray",
+      "email": "Felaray@gmail.com"
+    }
+  },
+  "host": "api.line.me",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": ["application/json"],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/v2/bot/message/push": {
+      "post": {
+        "summary": "Send Message To User",
+        "description": "Send a message to the associated user ID.",
+        "operationId": "SendMessage",
+        "parameters": [
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
```

Comment:
@felaray can you please clarify, why this has been marked as internal parameter?

Reply:
@Amjed-Ayoub I believe that users do not need to modify this parameter. If mark it as an internal parameter, users will not accidentally modify it.